code/modules/admin/admin_verbs.dm

```diff
@@ -5,19 +5,19 @@ GLOBAL_PROTECT(admin_verbs_default)
 /world/proc/AVerbsDefault()
 	return list(
 	/client/proc/deadmin,				/*destroys our own admin datum so we can play as a regular player*/
-	/client/proc/cmd_admin_say,			/*admin-only ooc chat*/
+	//client/proc/cmd_admin_say,			/*admin-only ooc chat*/
```

Comment:
if you're going to remove things actually remove them, dont comment them out

Reply:
I accidentally requested a PR on the wrong github lol. I meant to do it with my own fork and pushed to all.